pocs/camera/__init__.py

```diff
@@ -79,7 +82,9 @@ def kwargs_or_config(item, default=None):
 
     logger.debug("Camera config: {}".format(camera_info))
 
-    a_simulator = 'camera' in kwargs_or_config('simulator', default=list())
+    simulator_names = hardware.get_simulator_names(config=config, kwargs=kwargs)
+    logger.info(f'simulator_names = {", ".join(simulator_names)}')
```

Comment:
`info` seems a little much here too when coming from the camera since it is showing all values (not just camera ones).  Maybe an info if `a_simulator` is True? (although I thought we had that)

Reply:
Switched to debug.